src/lib/components/submit/SubmitSelect.svelte

```diff
@@ -0,0 +1,35 @@
+<script lang="ts">
+    import ExternalLink from '$lib/components/ui/ExternalLink.svelte';
+    import ProposalSelector from '$lib/components/proposals/ProposalSelector.svelte';
+	import SubmitContinue from '$lib/components/submit/SubmitContinue.svelte';
+	import { createEventDispatcher } from 'svelte';
+
+    export let proposalType: string | undefined;
+
+    const dispatch = createEventDispatcher();
+    const select = () => dispatch('pnwrkSelect', { proposalType });
```

Comment:
why do you dispatch the proposalType if the property is already known by the parent?